samples/OcelotKube/DownstreamService/Ocelot.Samples.OcelotKube.DownstreamService.csproj

```diff
@@ -1,16 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
-
   <PropertyGroup>
     <TargetFramework>net7.0</TargetFramework>
```

Comment:
Forgot to change!

Reply:
I can't change it, it must be checked before.

Reply:
Let's change later. Not an issue.